scripts/configurator/parser/cmake_var.py

```diff
@@ -0,0 +1,40 @@
+from typing import Dict, Any
+
+def generate_cmake_var(type :str, cmake_var_str):
```

Comment:
NAMING: This is a factory function, not a generator of any kind. Please don't use the `generate_` prefix. Typically `make` or `build` are used in the industry.